test/functional/mempool_compatibility.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
+# Copyright (c) 2017-2020 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test that mempool.dat is both backward and forward compatible between versions
```

Comment:
19153 I don't think we can do this...

In its current form the test is skipped when trying to run it locally. I would think we should at least add a comment or something indicating that this test can't actually be ran...